Documentation/using-corert/troubleshooting-corert.md

```diff
@@ -6,3 +6,4 @@ Sometimes you want to have more information how CoreRT work. ILC Compiler provid
 * `<IlcGenerateDgmlFile>true</IlcGenerateDgmlFile>`: Generates log files `ProjectName.codegen.dgml.xml` and `ProjectName.scan.dgml.xml` in DGML format.
 * `<IlcGenerateMapFile>true</IlcGenerateMapFile>`: Generates log files `ProjectName.map.xml` which describe layout of objects how CoreRT sees them.
 * `<IlcSingleThreaded>true</IlcSingleThreaded>`: Perform compilation on single thread.
+* `<IlcDumpIL>true</IlcDumpIL>`: Dump final IL after generatoin in the file `ProjectName.il`. This can be helpful when debugging IL transformation, like marshalling for example.
```

Comment:
```suggestion
* `<IlcDumpGeneratedIL>true</IlcDumpGeneratedIL>`: Dump IL for method bodies the compiler generated on the fly into `ProjectName.il`. This can be helpful when debugging IL generation - e.g. marshalling. The compiler maps debug information to this file, so it's possible to step in it and set breakpoints.
```